README.md

```diff
@@ -78,22 +86,22 @@ spec:
 
 ## What is a chaos engine?
 
-Refer 
-- https://github.com/litmuschaos/chaos-operator
-- https://docs.litmuschaos.io/docs/getstarted/
+The ChaosEngine is the main user-facing chaos custom resource with a namespace scope and is designed to hold information around how the chaos experiments are executed. It connects an application instance with one or more chaos experiments,
+
+For more details [refer](https://v1-docs.litmuschaos.io/docs/chaosengine/) 
 
 ## What is a litmus chaos chart and how can I use it?
 
-Refer 
-- https://github.com/litmuschaos/chaos-charts
-- https://hub.litmuschaos.io/
+Litmus chaos chart is the heart of litmus and contains the low-level execution information. They serve as off-the-shelf templates that one needs to "pull" (install on the cluster) before including them as part of a chaos run against any target applications (the binding being defined in the ChaosEngine).
+
+For more details refer [litmus docs](https://v1-docs.litmuschaos.io/docs/chaosexperiment/) and [hub](https://hub.litmuschaos.io/).
 
 ## What are the steps to get started?
 
 - Install Operator Components and RBAC and CRDs
 
   ```bash
-  kubectl apply -f https://litmuschaos.github.io/pages/litmus-operator-latest.yaml
+  kubectl apply -f https://litmuschaos.github.io/litmus/litmus-operator-v2.0.0.yaml
```

Comment:
Can we keep it latest so that we don't have an overhead to update it. Anyways it is master branch so it won't be an issue I guess. WDYT?